### ISSUE TITLE
Using MAP-I to fish out spatial co-ordinates  

### ISSUE THREAD

**Issue by 64BitAsura** [TUC-ProAut/VSA_Toolbox#3]
Imagine, I put together good face HD profile and keep it in my associative memory! Currently I am building query vector from a image with spatial co-ordinates of each feature bind to its HD. If now bind my face HD profile with query vector! Is there a way to extract spatial co-ordinates of features that matches with face profile HD? 

Sorry If I bothered with my questions 🙏

**Comment by scken** [comment]
Thank you for your question. 
Could you tell me what you mean by face HD profile and how is it constructed? 
So you have this vector, let's say F, and want to bind it with the query vector Q. The query vector Q results from bound spatial coordinates - features pairs. 
And now you want to interpret the result from F * Q in a way where feature-position pairs match F?

**Comment by 64BitAsura** [comment]
> Thank you for your question. Could you tell me what you mean by face HD profile and how is it constructed? So you have this vector, let's say F, and want to bind it with the query vector Q. The query vector Q results from bound spatial coordinates - features pairs. And now you want to interpret the result from F * Q in a way where feature-position pairs match F?

yes exactly! I am looking into feature position pairs in Q that matched with F! I tried a test via binding co-ordinates to Q, it seems possible but it is cluster of hits in Q. Then I need to do some clustering like k-mean! smaller features I can encode for interested patterns! Then I believe result will have good portions of hits to make cluster and easy to weed out any spares hits! Thats approach I am looking into now

Note: I am trying to avoid sliding window and selective search, because they are costly!

**Comment by scken** [comment]
Sorry for the late reply. 
My understanding of the problem is more like a comparison with your known item memory. Your item memory contains all possible feature-coordinate pairs, and your vector F is a subset of these combinations. If you want to find all the feature coordinates in F, all you need to do is compute the similarities between F and your item memory. The highest similarities indicate matches with your vector F. Based on your known spatial feature entries in your item memory, you can extract the spatial coordinates. 
The question is how exactly the vector F is constructed - I assumed it was a bundle of feature-coordinate pairs. However, if you want to extract information, you need to compare it with a known item memory to retrieve the spatial information. 

I hope this somehow answers the question.
